packages/manager/core/generator/app/conditional-templates/dashboard/[serviceName]/_layout.tsx.hbs

```diff
@@ -36,9 +36,13 @@ export default function DashboardPage() {
   const { error, isLoading, } = results[0];
 
   if (error) {
-    return <Errors
-        error={error.response}
-      />;
+    return (
+      <div className="flex justify-center">
```

Comment:
We are waiting for the updates on manager-components before modifying the components inside it to avoid having to much conflicts

Additionally, the _layout.tsx file will be removed since we don't use generouted anymore